docs/docs/agents/context.md

```diff
@@ -12,56 +12,66 @@ LangGraph provides **three** primary ways to supply context:
 
 | Type                                                                         | Description                                   | Mutable? | Lifetime                |
 |------------------------------------------------------------------------------|-----------------------------------------------|----------|-------------------------|
-| [**Config**](#config-static-context)                                         | data passed at the start of a run             | ❌        | per run                 |
+| [**Runtime Context**](#runtime-context)                                      | data passed at the start of a run             | ❌        | per run                 |
 | [**Short-term memory (State)**](#short-term-memory-mutable-context)          | dynamic data that can change during execution | ✅        | per run or conversation |
 | [**Long-term memory (Store)**](#long-term-memory-cross-conversation-context) | data that can be shared between conversations | ✅        | across conversations    |
 
 ## Provide runtime context
 
-### Config (static context)
+### Runtime Context
 
-Config is for immutable data like user metadata or API keys. Use
-when you have values that don't change mid-run.
+!!! note "`config['configurable']` -> `runtime.context`"
 
-Specify configuration using a key called **"configurable"** which is reserved
-for this purpose:
+    In LangGraph < v1.0, static runtime context was passed via the `config['configurable']` key, paired with a `config_schema` argument
+    to `StateGraph` or `Pregel`. This is now deprecated and will be removed in v2.0.
+
+    In LangGraph v1.0, we've introduced the [`Runtime`][langgraph.types.Runtime] object, which holds both static `context` and other
```

Comment:
Likely "we've" in the docs

As of LangGraph v1.0, the Runtime object is recommended to access static context and ...